plugins/modules/pagerduty_alert.py

```diff
@@ -53,11 +55,37 @@
             - 'triggered'
             - 'acknowledged'
             - 'resolved'
-    api_key:
+    api_version:
         type: str
         description:
-            - The pagerduty API key (readonly access), generated on the pagerduty site.
-        required: true
+            - The Api version we want to use to run the module.
+            - V1 is more limited with option we can provide to trigger incident
+            - V2 has more variables for example, severity, source, custom_detail,etc.
+        default: 'v1'
+        choices:
+            - 'v1'
+            - 'v2'
```

Comment:
New options need `version_added`.
```suggestion
            - 'v2'
        version_added: 6.6.0
```